queries.sql

```diff
@@ -8,3 +8,44 @@ SELECT name, date_of_birth FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 'Yes';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg>=10.4 AND weight_kg<=17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] Before COMMIT we have to review the changes in the column species and after too. Please add the lines `SELECT species FROM animals` to verify the changes.